### PR TITLE
feat: quick tab bar store page desktop (WIP)

### PR DESCRIPTION
**Pull request by hanggarHungry24**
ticket basecamp: https://3.basecamp.com/5190892/buckets/32591430/card_tables/cards/6106281862

### CODE REVIEW

File: helpers/abbreviationLabelPackageType.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+export function abbreviationLabelPackage(value: string): string {
```
Comment (afif-hh) [inline helpers/abbreviationLabelPackageType.ts:1]: ada `type` yg isinya handle ayce,pp, dll mas coba di cari

File: helpers/abbreviationLabelPackageType.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  let wording = "";
+  switch (value) {
+    case "ayce":
+      wording = "All You Can Eat";
```
Comment (afif-hh) [inline helpers/abbreviationLabelPackageType.ts:5]: ada helper nama nya `getPackageTypeFullName` mas

File: partial/restaurant_detail/sticky_header_navigation/RestaurantStickyHeaderDesktop.vue [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+});
+async function loadUseActiveLib() {
+  const activeScroll = () => {
+    return import("@/lib/vueActiveScroll");
+  };
+  const activeGalleryLib = (await retryablePromise(activeScroll)).default;
+  const useActive = activeGalleryLib();
+  return useActive;
+}
+const useActive = await loadUseActiveLib();
```
Comment (afif-hh) [inline partial/restaurant_detail/sticky_header_navigation/RestaurantStickyHeaderDesktop.vue:77]: ini coba di pindah ke onMounted nya component mas

**Review by afif-hh — CHANGES_REQUESTED** [review]

File: helpers/abbreviationLabelPackageType.ts (hunk 1 above)
Comment (hanggarHungry24) [reply helpers/abbreviationLabelPackageType.ts]: updated mas pake getPackageTypeFullName

File: helpers/abbreviationLabelPackageType.ts (hunk 2 above)
Comment (hanggarHungry24) [reply helpers/abbreviationLabelPackageType.ts]: updated mas pake getPackageTypeFullName

**Pull request closed without merging**